### DISCUSSION TITLE
Support for GD25LB64C

### DISCUSSION THREAD

**Discussion by aannenko** [nofeletru/UsbAsp-flash#100] [Q&A] [answered]
Hi,

GD25LB64C 1.8v connected over CH341A is recognized by AsProgrammer 2.0.4 as GD25LQ064C 1.8v and cannot be correctly written. @nofeletru, do you think support for it could be added? If not, what do you guys use to flash GD25LB64C?

GD25LB64C datasheet: https://www.gigadevice.com.cn/Public/Uploads/uploadfile/files/20220714/DS-00003-GD25LB64C-Rev1.8.pdf

**Comment by tvalk1** [comment]
I'd be glad if AsProgrammer supported GD25LB64C!

**Comment by aannenko** [comment, marked as answer]
So, AsProgrammer actually does support GD25LB64C in a form of GD25B64C (without L). So if you have 1.8v mod for your CH341a, use it and select GD25B64C - it should work just fine. I've just resurrected a laptop this way.

![IMG_20231003_114321](https://github.com/nofeletru/UsbAsp-flash/assets/4640265/9d9419bc-fff9-4a1b-a2cd-3a1d019ca39e)
